geopandas/tests/test_vectorized_api.py

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
Is there something specific tested in this file? (seems tests of actual usage of GeoSeries) 
In that case, they can be moved to one of the other files? (I can do that)